clarity-ext-scripts/clarity_ext_scripts/covid/parse_pcr.py

```diff
@@ -21,8 +21,17 @@ def execute(self):
                             target_name, output.well.alpha_num_key, file_name, output.name))
 
             ct = entry[CT_HEADER]
+
+            # add the measurement to the output artifact
             output.udf_map.force("CT", ct)
             self.context.update(output)
 
+            # also add the measurement to the original sample, where it should be understood
+            # as the latest measurement
+            original_sample = output.sample()
+            original_sample.udf_map.force("CT", ct)
```

Comment:
I'm considering to have the udf name like "CT latest", to indicate that this is a value that might be overwritten. What do you think about that?